File: app/src/main/java/com/taiwanlife/teamwalk/model/request/LoginRequest.java
package com.taiwanlife.teamwalk.model.request;

import androidx.annotation.NonNull;

public class LoginRequest {

    private String ticket;
    private String service;
    private String app_uuid;
    private String device_id;
    private String push_id;

    public LoginRequest(String ticket, String service, String app_uuid, String device_id, String push_id) {
        this.ticket = ticket;
        this.service = service;
        this.app_uuid = app_uuid;
        this.device_id = device_id;
        this.push_id = push_id;
    }

    public String getTicket() {
        return ticket;
    }

    public void setTicket(String ticket) {
        this.ticket = ticket;
    }

    public String getService() {
        return service;
    }

    public void setService(String service) {
        this.service = service;
    }

    public String getApp_uuid() {
        return app_uuid;
    }

    public void setApp_uuid(String app_uuid) {
        this.app_uuid = app_uuid;
    }

    public String getDevice_id() {
        return device_id;
    }

    public void setDevice_id(String device_id) {
        this.device_id = device_id;
    }

    public String getPush_id() {
        return push_id;
    }

    public void setPush_id(String push_id) {
        this.push_id = push_id;
    }

    @NonNull
    @Override
    public String toString() {
        return "LoginRequest{" +
                "ticket='" + ticket + '\'' +
                ", service='" + service + '\'' +
                ", app_uuid='" + app_uuid + '\'' +
                ", device_id='" + device_id + '\'' +
                ", push_id='" + push_id + '\'' +
                '}';
    }
}

File: app/src/main/java/com/taiwanlife/teamwalk/remote/request/api/LoginRequest.kt
package com.taiwanlife.teamwalk.remote.request.api

import com.google.gson.annotations.SerializedName
import com.taiwanlife.teamwalk.remote.request.BaseRequest

data class LoginRequest(
    /**
     * 登入者帳號
     */
    @SerializedName("appl_id")
    val applId: String,
    /**
     * CSSO Ticket資訊
     */
    @SerializedName("ticket")
    val ticket: String,
    /**
     * CSSO service
     */
    @SerializedName("service")
    val service: String,
    /**
     * app安裝後產生唯一識別碼
     */
    @SerializedName("app_uuid")
    val appUuid: String,
    /**
     * 裝置識別碼
     */
    @SerializedName("device_id")
    val deviceId: String,
    /**
     * FCM推播Token
     */
    @SerializedName("push_id")
    val pushId: String
) : BaseRequest()